### PR TITLE
Allow Let's Encrypt random sleep on renewal to be disabled

### PR DESCRIPTION
**Pull request by dlundgren**
This allows for self-hosted Bitwarden administrators to define that certbot should not sleep randomly on renewals. This was previously reported in #415, but LE has since introduced the `--no-random-sleep-on-renew` flag to prevent the delay.

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -142,13 +142,24 @@ function dockerPrune() {
         --filter="label!=com.bitwarden.project=setup"
 }
 
+function preventLetsEncryptRandomSleepOnRenew() {
+    if [ -f "${OUTPUT_DIR}/config.yml" ]
+    then
+        grep -E "^lets_encrypt_no_random_sleep_on_renew: true" "${OUTPUT_DIR}/config.yml" 2>&1 >/dev/null
+        if [ "$?" -eq 0 ]
```
Comment (chadmiller) [inline scripts/run.sh:149]: No need to run grep separately from test ("["). You can just

```
if grep -E "^lets_encrypt"...; then
```

Also, "-q" stops grep stdout.

Comment (dlundgren) [reply scripts/run.sh]: Thanks for the review! I've updated with your suggestions.

**Review by joseph-flinn — CHANGES_REQUESTED** [review]
Adding a config value to the `config.yml` file will require updates to the `Setup`. If there are values in that file that the `Setup` project are not expecting, it will break on any `./bitwarden.sh update` commands

**Comment by dlundgren** [comment]
I quite thoroughly messed this PR up, I'll create a new one off of a fresh master.

**Pull request closed without merging**